client/deposits/index.tsx

```diff
@@ -10,13 +10,16 @@ import { useDispatch } from '@wordpress/data';
  * Internal dependencies.
  */
 import Page from 'components/page';
+import interpolateComponents from '@automattic/interpolate-components';
+import { __ } from '@wordpress/i18n';
 import { TestModeNotice } from 'components/test-mode-notice';
 import BannerNotice from 'components/banner-notice';
 import DepositSchedule from 'components/deposits-overview/deposit-schedule';
 import { useAllDepositsOverviews } from 'data';
 import { useSettings } from 'wcpay/data';
 import DepositsList from './list';
 import { hasAutomaticScheduledDeposits } from 'wcpay/deposits/utils';
+import { ExternalLink } from '@wordpress/components';
```

Comment:
Nitpick: Let's keep external imports together.

Why?
It makes it easier to identify and manage dependencies in a project. By grouping all external imports together, you can quickly see which packages your code is dependent on. This can be especially useful when you need to update or remove a package, as you can easily identify all the parts of your code that rely on it.

Reply:
Moved to External imports section.